addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -31,7 +31,8 @@ export class MessageReactionList extends Component {
         const personNames = reaction.personas
             .slice(0, 3)
             .map((persona) => this.props.message.getPersonaName(persona));
-        const shortcode = this.store.emojiLoader.loaded?.emojiValueToShortcodes?.[emoji][0] ?? "?";
+        const shortcode =
+            this.store.emojiLoader.loaded?.emojiValueToShortcodes?.[emoji]?.[0] ?? "?";
```

Comment:
Shouldn't it fallback to `emoji` (the content itself) rather than `?` when it is not set?

Reply:
It shows the big emoji already:
<img width="258" alt="Screenshot 2025-06-11 at 16 57 42" src="https://github.com/user-attachments/assets/9423f779-ca7f-48a7-bb8c-0a47a9df8696" />
However we could just remove the shortcode entirely and just show textually "reacted by X, Y and Z".